bundles/AdminBundle/src/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1900,6 +1902,26 @@ public function getBrickUsagesAction(Request $request): Response
         return $this->adminJson($result);
     }
 
+    /**
+     * @Route("/get-select-options-usages", name="getselectoptionsusages", methods={"GET"})
+     */
```

Comment:
```suggestion
     #[Route("/get-select-options-usages", name: "getselectoptionsusages", methods: ["GET])]
```

Reply:
I think as of Pimcore 11 attributes should be used.

Reply:
It made sense to me to keep it consistent with the other routes in this controller. But I guess the difference isn't that big.